check/bench_test.go

```diff
@@ -0,0 +1,143 @@
+package check
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+
+const check = `---
+controls:
+id: 1
+text: "Master Node Security Configuration"
+type: "master"
+groups:
+- id: 1.1
+  text: "API Server"
+  checks:
+    - id: 1.1.1
+      text: "Ensure that the --allow-privileged argument is set to false (Scored)"
+      audittype: "wrong_type"
+      audit: "ps -ef | grep $apiserverbin | grep -v grep"
+      tests:
+        test_items:
+        - flag: "allow-privileged"
+          compare:
+            op: eq
+            value: false
+          set: true
+      remediation: "Edit the $apiserverconf file on the master node and set 
+              the KUBE_ALLOW_PRIV parameter to \"--allow-privileged=false\""
+      scored: true
+`
+
+
+func Test_bench_RegisterAuditType_Already_Registered(t *testing.T) {
+
+	bench := NewBench()
+
+	if err := bench.RegisterAuditType(TypeAudit, func() interface{} {
+		return Audit("test")
+	})	; err != nil{
+
+		t.Errorf("Failed to register new Audit type")
+		return
+	}
+	if err := bench.RegisterAuditType(TypeAudit, func() interface{} {
+		return Audit("test")
+	})	; err == nil{
+
+		t.Errorf("RegisterAuditType failed on duplicate types")
```

Comment:
Better error text might be “RegisterAuditType should have failed on duplicate types”